utils.py

```diff
@@ -1,6 +1,14 @@
-"""
-Utilities module for the app.
-"""
+from flask import abort
+from http.client import BAD_REQUEST
+
+
+def check_required_fields(required_fields, request_data):
```

Comment:
visto che è una funzione di utilità che dovrebbe essere usata in più posti, è *obbligatorio* avere una docstring che descriva cosa fa la funzione e i parametri in input / output